specification/app/resource-manager/Microsoft.App/stable/2022-05-01/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -159,6 +176,23 @@
                 ]
               }
             ],
+            "initContainers": [
```

Comment:
Update `specification\app\resource-manager\Microsoft.App\preview\2022-06-01-preview\ContainerApps.json` for the schema